### ISSUE TITLE
Remove markdown log file support

### ISSUE THREAD

**Issue by spbrogan** [tianocore/edk2-pytool-extensions#289]
**Is your feature request related to a problem? Please describe.**
Remove the markdown logger.  This logger isn't being used and just wastes time in the logging path. 

**Describe the solution you'd like**
Remove it

**Comment by Javagedes** [comment]
@spbrogan #373 has been created for this. Do you want the actual logging removed from edk2toollib?

**Comment by spbrogan** [comment]
Yes.  I think we should.  It had a lot of tentacles into logging so getting better aligned on python standard logging would be good.


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
